### ISSUE TITLE
Importing combined_from broken in 1.032

### ISSUE THREAD

**Issue by perlpunk** [briandfoy/test-output#8]
It seems `combined_from` is not exported by default anymore. Importing all functions works:
```
use Test::More tests => 1;
use Test::Output qw(:functions);
say Test::Output->VERSION;
my $out = combined_from { say "hello" };
like $out, qr{hello};
__END__
1..1
1.032
ok 1
```
But not by default:
```
use Test::More tests => 1;
use Test::Output;         
say Test::Output->VERSION;
my $out = combined_from { say "hello" };
like $out, qr{hello};
__END__
1..1
1.032
hello
Can't locate object method "combined_from" via package "1" (perhaps you forgot to load "1"?) at -e line 5.
# Looks like your test exited with 255 before it could output anything.
```
It worked in 1.031:
```
use Test::More tests => 1;
use Test::Output;
say Test::Output->VERSION;
my $out = combined_from { say "hello" };
like $out, qr{hello};
__END__
1..1
1.031
ok 1
```

Other functions are still working:
```
use Test::More tests => 1;
use Test::Output;
say Test::Output->VERSION;
my $out = combined_like { say "hello" } qr{hello}, "label"
__END__
1..1
1.032
ok 1 - label
```

**Comment by briandfoy** [comment]
This is a very odd situation. Reading the code and the module, I think this is the correct behavior. However, it does work in 1.031. And, none of the export code has changed in 15 years.

* The docs say that all tests are exported by default. `combined_from` is not a test and is not included in the list of `tests` in the export list.

```
    tests => [
46
      qw(
47
        output_is output_isnt output_like output_unlike
48
        stderr_is stderr_isnt stderr_like stderr_unlike
49
        stdout_is stdout_isnt stdout_like stdout_unlike
50
        combined_is combined_isnt combined_like combined_unlike
51
        )
52
    ],
53
    default => [ '-tests' ],
54
  },
```

**Comment by briandfoy** [comment]
Oh, I'm so wrong here. The bisect doesn't work because I released from an unmerged branch. Major screw up on my part. But, that has now been in the wild for four years, so I need to merge and re-release.

I see in the original issue that you fixed the problem with explicit imports, which is the right fix at the application level. If you know what you want, import that. But still, I broke your code doing something totally stupid that shouldn't have happened.
